static/js/settings_org.js

```diff
@@ -320,18 +320,14 @@ function _set_up() {
         _.each(property_types.settings, function (v, k) {
             property_type_status_element(k).hide();
         });
-        var name_status = $("#admin-realm-name-status").expectOne();
-        var waiting_period_threshold_status = $("#admin-realm-waiting-period-threshold-status").expectOne();
-        name_status.hide();
-        waiting_period_threshold_status.hide();
+        var status = $(".settings-section.show .alert").eq(0);
```

Comment:
Why are you using `.eq(0)` rather than our traditional `expectOne()`.  The latter works with our unit tests, and I think this is why the node tests are failing.